lib/Doctrine/Template/I18n.php

```diff
@@ -32,6 +32,11 @@
  */
 class Doctrine_Template_I18n extends Doctrine_Template
 {
+    /**
+     * @var Doctrine_I18n $_plugin
+     */
+    protected $_plugin;
```

Comment:
Just so we can document what `$_plugin` is in this class (visibility not changed since Doctrine_Template defines it the same).